akka-stream/src/main/boilerplate/akka/stream/javadsl/FlowCreate.scala.template

```diff
@@ -13,13 +13,32 @@ trait FlowCreate {
   import language.implicitConversions
   private implicit def p[A, B](pair: Pair[A, B]): (A, B) = pair.first -> pair.second
 
+  /**
+   * Creates a Flow by passing a [[FlowGraph.Builder]] to the given create function.
```

Comment:
seen on Sources and Sinks: please also mention the meaning of the expected `Pair<Inlet<I>, Outlet<O>>` return value